src/appendices/spring/thymeleaf.rst

```diff
@@ -0,0 +1,132 @@
+.. _thymeleaf-commands:
+
+Thymeleaf Commands
+==================
+
+Thymeleaf is a template engine, and Thymeleaf templates make up the views of our MVC application.
+In order to create our templates, we use attributes to pass data from the model, through the controllers, and to our views.
+
+Attributes
+----------
+
+Thymeleaf has many attributes that we could use. In this book, we focus on a few key ones.
+
+Displaying Data
+^^^^^^^^^^^^^^^
+
+``th:text`` dynamically populates the contents of an HTML element.
+
+.. sourcecode:: html
+
+   <p th:text = "Hello World!"></p>
+
+If we want to pull in a value of a variable from the controller, we can use the **variable expressions** syntax, ``${}``.
+Let's say we want to pull in the value of a variable called ``hello``.
+
+.. sourcecode:: html
+
+   <p th:text = "${hello}"></p>
+
+Conditionally Displaying Data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to conditionally display data, we need to use ``th:if`` and ``th:unless``.
+``th:if`` will display the value of the element if the expression evaluates to true. If the expression evaluates to false, then the value of the element is NOT displayed.
+``th:unless`` will display the value of the element if the expression evaluates to false. If the expression evaluates to true, then the value of the element is NOT displayed.
```

Comment:
Should we highlight ``true`` and ``false``?

Reply:
Done!